etc/kayobe/pulp.yml

```diff
@@ -10,6 +10,9 @@ pulp_url: "http://{{ admin_oc_net_name | net_ip(groups['seed'][0]) }}:80"
 pulp_username: admin
 pulp_password: "{{ secrets_pulp_password }}"
 
+# Proxy to use when adding remotes.
+pulp_proxy_url: "{{ https_proxy | default(omit) }}"
```

Comment:
In the case where we have a squid proxy on the seed, this would probably not make sense. Which case is more common? Corporate or squid?

Reply:
Should we start with `pulp_proxy_url: "{{ omit }}"` so you have to explicitly define it? We can refine the default at a later time?

Reply:
:+1: